Tests/StencilSwiftKitTests/TestsHelper.swift

```diff
@@ -42,7 +45,12 @@ func diff(_ result: String, _ expected: String) -> String? {
     }
     let lhsNum = addLineNumbers(slice(lhsLines, 4)).joined(separator: "\n")
     let rhsNum = addLineNumbers(slice(rhsLines, 4)).joined(separator: "\n")
-    return "\(msgColor)Mismatch at line \(badLineIdx)\(reset)\n>>>>>> result\n\(lhsNum)\n======\n\(rhsNum)\n<<<<<< expected"
+    return "\(msgColor)Mismatch at line \(badLineIdx)\(reset)\n" +
+      ">>>>>> result\n" +
+      "\(lhsNum)\n" +
+      "======\n" +
+      "\(rhsNum)\n" +
+      "<<<<<< expected"
```

Comment:
I think an array of strings that gets `join`'d is more efficient that successive string concatenation… and compiles faster (because `+` is so overloaded) too